src/app/Http/Controllers/API/PresentationController.php

```diff
@@ -68,7 +68,7 @@ public function store(Request $request, User $user)
      */
     public function show(Presentation $presentation)
     {
-        //
+        return new PresentationResource($presentation);
```

Comment:
"I can't understand, is this the language of the gods?"

Reply:
Nope, it's the language of Satan